### ISSUE TITLE
Archive this repo

### ISSUE THREAD

**Issue by tkurki** [sailoog/openplotter#274]
If this repo is dead Github supports archiving it, that will for example disable issue creation.

**Comment by tkurki** [comment]
Consider also adding a pointer to the home in title or readme.

**Comment by sailoog** [comment]
Yes, that should be done also for different documentation versions. Thanks for reminder.
